docs/serving/available-steps.md

```diff
@@ -73,26 +73,30 @@ FeaeturesetValidator | Y      | N     | Y      |
-| [mlrun.datastore.TSBDTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.TSDBTarget) | Writes incoming events to TSDB table. | 
+| Class name                                                                                                                | Description                                                                                                  | storey | spark | pandas |
+| --------------------------------------------------                                                                        | -------------------------------------------------------                                                      | ---    | ---   | ---    |
+| [mlrun.datastore.CSVTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.CSVTarget)                    | Writes events to a CSV file.                                                                                 | Y      | Y     | Y      |
+| [mlrun.datastore.NoSqlTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.NoSqlTarget)                | Persists the data in V3IO *table* to its associated storage by key.                                          | Y      | Y     | Y      |
+| [mlrun.datastore.RedisNoSqlTarget]                                                                                        | Persists the data in Redis *table* to its associated storage by key.                                         | Y      | Y     | N      |
+| [mlrun.datastore.SqlTarget]                                                                                               | Persists the data in SQL *table* to its associated storage by key.                                           | Y      | N     | Y      |
+| [mlrun.datastore.ParquetTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.ParquetTarget)            | The Parquet target storage driver, used to materialize feature set/vector data into parquet files.           | Y      | Y     | Y      |
+| [mlrun.datastore.StreamTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.StreamTarget)              | Writes all incoming events into a V3IO stream.                                                               | Y      | N     | N      |
+| [mlrun.datastore.KafkaTarget]                                                                                             | Writes all incoming events into a Kafka stream.                                                              | Y      | N     | N      |
+| [storey.transformations.ToDataFrame](https://storey.readthedocs.io/en/latest/api.html#storey.transformations.ToDataFrame) | Create pandas data frame from events. Can appear in the middle of the flow, as opposed to ReduceToDataFrame. | Y      | N     | N      |
```

Comment:
I know it was there before, but it's kind of strange that this references a step that is not documented on the list (`ReduceToDataFrame`).

Reply:
removed the reference